crates/sound/src/audio_backends/pipewire.rs

```diff
@@ -158,12 +75,12 @@ impl PwBackend {
     pub fn new(stream_params: Arc<RwLock<Vec<Stream>>>) -> Self {
         pw::init();
 
-        let thread_loop = Rc::new(PwThreadLoop::new(Some("Pipewire thread loop")).unwrap());
-        let get_loop = thread_loop.get_loop();
+        #[allow(unused_unsafe)]
```

Comment:
I tried to remove this and my `clippy 0.1.72 (5680fa1 2023-08-23)` is not printing any warning.

What version of clippy do you see the warning with, or is it old code?

Reply:
Ah, that was old code thanks :) , I am using same clippy 0.1.72